### ISSUE TITLE
engineSetModelLODDistance doesn't working

### ISSUE THREAD

**Issue by FileEX** [multitheftauto/mtasa-blue#2539]
**Describe the bug**
engineSetModelLODDistance don't working but returns true.

**To reproduce**
```lua
engineSetModelLODDistance(5235, 500) -- true
local dist = engineGetModelLODDistance(5235) -- 325
print(dist)
```
It's occurs for any models id

Second code
```lua
local readData = fileRead(data, fileGetSize(data));
engineSetModelLODDistance(models[k], tonumber(readData));

print('SET LOD TO ', readData, 'ACTUAL LOD IS', engineGetModelLODDistance(models[k]));
```

**Expected behaviour**
LOD should be changed or shouldn't returns TRUE.

**Screenshots**
![image](https://user-images.githubusercontent.com/26044224/152462508-25241021-a527-45ec-900e-23c74294a134.png)
![image](https://user-images.githubusercontent.com/26044224/152463318-ef0a30c4-4f82-402e-81e1-7d972fdf9cb3.png)

**Version**
1.5.9

**Additional context**
This bug occurs only if new LOD distance is bigger than default.

**Comment by Vadya963** [comment]
However, there is a general draw distance limit of 300 units. So engineSetModelLODDistance(1337,400) will mean model 1337 will be visible up to a distance of 300 units no matter what the 'Video' tab says.

**Comment by FileEX** [comment]
Yeah, but i think if distance > 300 then engineSetModelLODDistance shouldn't returns true?
